### PR TITLE
test: add integration tests against Postgres.js

### PR DESCRIPTION
**Pull request by hauleth**
## What kind of change does this PR introduce?

Add integration tests against Postgres.js Node library.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -78,20 +83,31 @@
...
+                package = pkgs.postgresql_15;
                 initialScript = ''
                   ${builtins.readFile ./dev/postgres/00-setup.sql}
 
                   CREATE USER postgres SUPERUSER PASSWORD 'postgres';
                 '';
                 listen_addresses = "127.0.0.1";
                 port = 6432;
+                settings = {
+                  max_prepared_transactions = 262143;
```
Comment (hauleth) [inline flake.nix:95]: It is maximal possible value, in tests I have set it to `2000` as it doesn't matter really as long as it is greater than `0` I think.

File: .github/workflows/elixir.yml [hunk 2]
```diff
@@ -127,6 +127,54 @@ jobs:
...
+    needs: [deps]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Elixir
+        id: beam
+        uses: erlef/setup-beam@v1
+        with:
+          otp-version: '25.3.2.7'
+          elixir-version: '1.14.5'
```
Comment (abc3) [inline .github/workflows/elixir.yml:142]: wondering if we want to put these versions in the env vars

Comment (hauleth) [reply .github/workflows/elixir.yml:142]: Could be useful, but as a separate PR. There I simply copied what we have in other steps.

File: test/integration/js/postgres/copy.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+1	2	3
+4	5	6
```
Comment (abc3) [inline test/integration/js/postgres/copy.csv:2]: what does it do?

Comment (hauleth) [reply test/integration/js/postgres/copy.csv:2]: No idea, it is copied as is from `postgres.js` test folder. I haven't really checked if that is used anywhere in test suite.

Comment (abc3) [reply test/integration/js/postgres/copy.csv:2]: let's try to run it without this file :D

Comment (hauleth) [reply test/integration/js/postgres/copy.csv:2]: I have found that it is used in one test:
```js
t('Copy from file', async() => {
  await sql`create table test (x int, y int, z int)`
  await new Promise(async r => fs
    .createReadStream(rel('copy.csv'))
    .pipe(await sql`copy test from stdin`.writable())
    .on('finish', r)
  )

  return [
    JSON.stringify(await sql`select * from test`),
    '[{"x":1,"y":2,"z":3},{"x":4,"y":5,"z":6}]',
    await sql`drop table test`
  ]
})
```

**Pull request merged**